fastlane/Fastfile

```diff
@@ -418,7 +418,7 @@ platform :ios do
     en_lproj = 'WordPress/Resources/en.lproj'
     ios_generate_strings_file_from_code(
       paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
-      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/Classes/Utility/AppLocalizedString.swift'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/**', '**/AppLocalizedString.swift'],
```

Comment:
Excluding all the test files. After all, we don't want to pick up any `NSLocalizedString` (and their `comment:`) used in tests for the strings we send to translators.